Tools/cases_generator/stack.py

```diff
@@ -3,6 +3,7 @@
 from dataclasses import dataclass
 from cwriter import CWriter
 
+UNUSED = {"unused", "__unused_"}
```

Comment:
Why, what's wrong with "unused" as a name.

Reply:
I start mangling names later. So I need to treat both as unused.

Reply:
But you aren't doing it yet, and it isn't tested. So can we leave this be for now?